recipes/igraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "igraph" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://igraph.org/nightly/get/c/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }} 
+  patches:
+    - b215195214abb8355121a8430c86df04e1f00eae.patch  # fixes igraph/igraph#906
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - gmp
+    - libxml2
+    - pkg-config
+  run:
+    - gmp 
+    - libxml2
+
+test:
+  files:
+    - igraph_test.c
+  requires:
+    - pkg-config
+
+about:
+  home: http://igraph.org
+  license: GPL2
+  license_file: COPYING
+  summary: "igraph is a collection of network analysis tools with the emphasis on efficiency, portabiliy and ease of use. igraph is open source and free. igraph can be programmed in R, Python and C/C++."
```

Comment:
Could we shorten this line? Info of this length can go in `description`, but should be [wrapped]( https://github.com/conda-forge/staged-recipes/blob/99c66d02f87f1bcc5eaaa7b835b833430a9cfc19/recipes/example/meta.yaml#L58-L63 ).